src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -423,6 +426,7 @@ typedef struct
 typedef struct
 {
     double* ValeursHorairesDeDefaillancePositive;
+    double* ValeursHorairesDENS; // adq patch DENS
```

Comment:
domestic unsupplied energy (comment)

Reply:
fixed in https://github.com/rte-i/Antares_Simulator/pull/30